### PR TITLE
HTML INPUT CHANGES - CSS ADDITIONS - JAVASCRIPT ADDITIONS

### PR DESCRIPTION
**Pull request by jpatterson933**
TLDR; I  found it difficult to calculate our free hours per day using the drop down lists so I created and added in checkboxes and used those values to calculate and display total free hours in jumbotron.

### New Checkboxes
The checkboxes, when checked, will store a value into a variable called sel. Using a for loop through that variable, the values will then be turned into numbers and added together using this line of code in the javascript on line 13 `total += sel[i] << 0;`

### Display Hours
After the for loop, there is an if statement that basically says, if the amount of time you spend per day doing the activities that were checked is greater than 24 it will display fuggedaboutit. Else, it will display the total free hours.   
 ```
 if (total > 24) {
        $('#available-free-time').text("Fuggedaboutit!");
    } else {
        $("#available-free-time").text("Total Free Time " + (24 - total) + " hours!");
    };
```

### How total hours was calculated
Total free hours is calculated on line 24 using this line of text `$("#available-free-time").text("Total Free Time " + (24 - total) + " hours!");` Basically, take 24 hours in the day and subtract the users checkbox answers.

### Additional CSS elements
Finally, in the CSS file on lines 168-188 this is where the new styling was added for the checkboxes.

### <------------------------------------------------- --------------------------------->
I wrapped all new elements in HTML, CSS, and JAVASCRIPT like this so you can find them easily. I also added more comments
### <------------------------------------- ------------------------------------>

> Whether you want to uncover the secrets of the universe, or you just want to pursue a career in the 21st century, basic computer programming is an essential skill to learn.
> 
> Stephen Hawking

### CODE REVIEW

**Review by anthonyapicella — APPROVED** [review]
# NOOICE
